src/checks/stylelingConfigIsValid.ts

```diff
@@ -0,0 +1,91 @@
+import { promises as fs } from 'node:fs';
+import { createRequire } from 'node:module';
+import { execaCommand } from 'execa';
+import readdirp from 'readdirp';
+import semver from 'semver';
+
+const require = createRequire(`${process.cwd()}/`);
+
+export const title = 'Stylelint Config is latest version';
+
+export default async function stylelintConfigIsValid() {
+  const { stdout: remoteVersion } = await execaCommand(
+    'npm show stylelint-config-upleveled version',
+  );
+
+  let localVersion;
+
+  try {
+    const stylelintConfigPackageJsonPath = require.resolve(
+      'stylelint-config-upleveled/package.json',
+    );
+
+    localVersion = JSON.parse(
+      await fs.readFile(stylelintConfigPackageJsonPath, 'utf-8'),
+    ).version;
+  } catch (err) {}
+
+  if (typeof localVersion === 'undefined') {
+    throw new Error(
```

Comment:
oh I just realized this is going to be a bug for the Node.js projects:

```js
if ('react-scripts' in projectDependencies || 'next' in projectDependencies) {
  newDevDependenciesToInstall.push('stylelint', 'stylelint-config-upleveled');
}
```

https://github.com/upleveled/eslint-config-upleveled/blob/df447996c8a01039f84489f31161e316be6cddd8/bin/install.js#L51-L53

Reply:
098ee24